tests/testsuite/cargo_command.rs

```diff
@@ -42,6 +42,25 @@ fn list_aliases_with_descriptions() {
         .run();
 }
 
+#[cargo_test]
+fn list_custom_aliases_with_descriptions() {
+    let p = project_in_home("proj")
+        .file(
+            &paths::home().join(".cargo").join("config"),
+            r#"
+            [alias]
+            myaliasstr = "foo --bar"
+            myaliasvec = ["foo", "--bar"]
+        "#,
+        )
+        .build();
+
+    p.cargo("--list")
+        .with_stdout_contains("    myaliasstr           foo --bar")
+        .with_stdout_contains("    myaliasvec           foo --bar")
+        .run();
+}
+
```

Comment:
nice! Lgtm!

Reply:
Thank you so much!  On projects where I _am_ a maintainer I really appreciate other contributors looking at PRs, and I bet the cargo maintainers do too.